Tensile/TensileCreateLibrary.py

```diff
@@ -797,10 +806,17 @@ def writeKernels(
         itertools.repeat(kernelWriterSource),
         itertools.repeat(kernelWriterAssembly),
     )
-    results = Common.ParallelMap(processKernelSource, kIter, "Generating kernels")
+    results = Common.ParallelMap(processKernelSource, list(kIter), "Generating kernels")
+
     filterProcessingErrors(kernels, solutions, results, params["PrintLevel"], errorTolerant)
 
-    kernelFiles, kernelsWithBuildErrors = buildKernelSourceAndHeaderFiles(results, outputPath)
+    kernelFiles, kernelsWithBuildErrors = buildKernelSourceAndHeaderFiles(
+        results,
+        outputPath,
+        params["LazyLibraryLoading"],
+        params["MergeFiles"],
+        params["NumMergedFiles"],
+    )
```

Comment:
Move all details relating to refactors of `buildKernelSourceAndHeaderFiles` to a separate PR. Not in scope.